openapi/api.yml

```diff
@@ -2788,7 +2797,7 @@ definitions:
   WalletResponse:
     type: object
     required:
-      - data
+        - data
```

Comment:
@infantesimone Is this alignment required?

Reply:
thanks, that's a mistake.